pkgs/top-level/all-packages.nix

```diff
@@ -15086,6 +15086,10 @@ with pkgs;
   inherit (haxePackages) hxcpp;
 
   hop = callPackage ../development/compilers/hop { };
+  
+  hop-cli = callPackage ../tools/admin/hop-cli {
```

Comment:
```suggestion
  hop = callPackage ../development/compilers/hop { };
 
  hop-cli = callPackage ../tools/admin/hop-cli {
```
There is a single space causing the EditorConfig check to fail.

Reply:
Ack. Thanks for catching that :)